meson.build

```diff
@@ -52,10 +52,11 @@ configure_file(
   configuration: wrapper_config,
 )
 
+dep_gtk4 = dependency('gtk4', version: '>= 4.10')
+
 # Check for some runtime dependencies so they are displayed with an "NO"
 # when building which could give the user an idea what could be missing.
 dep_glib = dependency('glib-2.0', required: false)
-dep_gtk = dependency('gtk+-3.0', required: false)
```

Comment:
I'm pretty sure GTK3 is no longer a dependency! Also, I think we want GtkSourceView 5, but one thing at a time.